src/libstore/globals.hh

```diff
@@ -678,26 +678,76 @@ public:
         Strings{"https://cache.nixos.org/"},
         "substituters",
         R"(
-          A list of URLs of substituters, separated by whitespace. Substituters
-          are tried based on their Priority value, which each substituter can set
-          independently. Lower value means higher priority.
-          The default is `https://cache.nixos.org`, with a Priority of 40.
+          A list of URLs of substituters, separated by whitespace.
+
+          Substituters are tried based on their priority value, which each substituter can set on their own and independently.
+          Lower value means higher priority.
+          The default is `https://cache.nixos.org`, with a priority of 40.
+
+          It is possible to override a substituter's priority, for example:
+
+          ```
+          substituters = https://cache.nixos.org?priority=10
+          ```
 
           At least one of the following conditions must be met for Nix to use
           a substituter:
 
           - the substituter is in the [`trusted-substituters`](#conf-trusted-substituters) list
           - the user calling Nix is in the [`trusted-users`](#conf-trusted-users) list
 
-          In addition, each store path should be trusted as described
-          in [`trusted-public-keys`](#conf-trusted-public-keys)
+          In addition, each store path should be trusted as described in [`trusted-public-keys`](#conf-trusted-public-keys).
```

Comment:
I don't think this is a good place for substituter-specific information. It'd be better to refer to sections about each specific substituter type. This isn't scalable.

Reply:
I suppose this is addressed by https://github.com/NixOS/nix/pull/8084/